manim/mobject/number_line.py

```diff
@@ -50,6 +94,53 @@ def __init__(
         x_max=config["frame_x_radius"],
         **kwargs
     ):
+        """
+
+        Parameters
+        ----------
+        color : :class:`~.Colors`, optional
+            the color for the number line, by default LIGHT_GREY
+        unit_size : :class:`int`, optional
+            the unit size, by default 1
+        width : :class:`float`, optional
+            the width of the number line, by default None
+        include_ticks : :class:`bool`, optional
```

Comment:
it should be optional only when the default value is None, otherwise no.

Reply:
Sorry, am not sure what is it that you are suggesting here.

Reply:
same https://github.com/ManimCommunity/manim/pull/1064#discussion_r584455202